redisdb/check.py

```diff
@@ -239,7 +239,11 @@ def _check_db(self, instance, custom_tags=None):
             else:
                 l_tags = list(tags)
                 for key in key_list:
-                    key_type = conn.type(key)
+                    try:
+                        key_type = conn.type(key)
+                    except redis.ResponseError:
```

Comment:
Could the ResponseError be due to anything other than a `MOVED` key? Should we perhaps check that `MOVED` is in the ResponseError, and the continue? Or do you consider we should always `continue` upon error?

Reply:
TBH, I don't know that much about redis. But it seemed that whether the failure was from the being moved, not existing, or whatever other response error might get raised, it was better to keep trying to process any other keys rather than just bail out. I was also trying to keep it simple. 

I fixed the indentation issue.

Reply:
Also, sorta a n00b when it comes to git/github and this is the my first PR for DD, so not sure how to update the manifest and changelog, or to what.

Reply:
Thanks for the contribution, @chadharvey. https://docs.datadoghq.com/guides/integration_sdk/ has some additional information about the manifest/CHANGELOG. Additionally, we have Office Hours tomorrow if you'd like real-time assistance on this.  https://github.com/DataDog/dd-agent/wiki/Community-Office-Hours